polaris-react/src/components/Columns/Columns.tsx

```diff
@@ -51,7 +60,7 @@ export function Columns({columns, children, spacing}: ColumnsProps) {
     '--pc-columns-space-xl': spacing?.xl
       ? `var(--p-space-${spacing?.xl})`
       : undefined,
-  } as React.CSSProperties;
+  } as unknown as React.CSSProperties;
```

Comment:
Might have to collab with @aaronccasanova the TS 🧙‍♂️ on this. Fine for now though and we can iterate as we add similar functionality to other responsive components

Reply:
I'm not blocked by this, so all good!